### DISCUSSION TITLE
Find my done katas

### DISCUSSION THREAD

**Discussion by igormiliko** [codewars/codewars.com#2399] [Q&A] [answered]
It's a simple question... Where I can find my done katas? kkkkkk

**Comment by kazk** [comment, marked as answer]
In kata tab on your profile. https://www.codewars.com/users/igormiliko/completed

**Comment by igormiliko** [comment]
Thank you very much man, god blessed you.

Em qua., 9 de jun. de 2021 às 21:42, kazk ***@***.***>
escreveu:
